payments/card/after-payment.md

```diff
@@ -635,3 +633,4 @@ next_href="other-features" next_title="Next: Other Features" %}
 [capture]: /payments/credit-card/after-payment/#Capture
 [operations]: /payments/credit-card/other-features/#operations
 [reversal]: /payments/credit-card/after-payment/#reversals
+[seamless-view]: /payments/card/seamless-view
```

Comment:
I'm pretty sure the `prev_href` in `iterator` resolves to a path relative to the current file and not to a Markdown link. This footnote link should therefore not be needed.

```suggestion
```